test/mocha/10-test.js

```diff
@@ -45,3 +45,72 @@ describe('bedrock-jsonld-document-loader', () => {
     });
   });
 });
+
+describe('httpClientHandler', () => {
+  it('properly gets document', async () => {
+    let result;
+    let error;
+    const documentUrl = 'https://httpbin.org/json';
+
+    try {
+      result = await httpClientHandler.get({url: documentUrl});
+    } catch(e) {
+      error = e;
+    }
+    assertNoError(error);
+    should.exist(result);
+    result.should.eql({
+      contextUrl: null,
+      document: {
```

Comment:
Maybe pull this document out into a const.  Would make it easier to read the check here for document loader fields.

Is this a constant at httpbin?  Or does it change over time?

Reply:
The response data does seem to be a constant https://github.com/postmanlabs/httpbin/blob/f8ec666b4d1b654e4ff6aedd356f510dcac09f83/httpbin/core.py#L1761 and looks like it hasn't changed for years.